src/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoader.java

```diff
@@ -2926,40 +2927,16 @@ public Problem loadCCSProblemFiles(IInternalContest contest, String dataFileBase
 
         if (inputFileNames.length == answerFileNames.length) {
 
-            Arrays.sort(inputFileNames);
-            Arrays.sort(answerFileNames);
-
             ArrayList<SerializedFile> dataFiles = new ArrayList<SerializedFile>();
             ArrayList<SerializedFile> answerFiles = new ArrayList<SerializedFile>();
             
             if (loadSamples) {
+                // Load all sample files first, before loading files from sample dir.
```

Comment:
This comment doesn't seem to make sense.  What does it mean to "load all sample files first",  but "before loading files from sample dir"?   Loading the sample files is going to load files from the sample dir, isn't it?